OracleGoldenGate/21c/README.md

```diff
@@ -66,8 +66,9 @@ $ docker run \
     -e OGG_ADMIN=<admin user name> \
     -e OGG_ADMIN_PWD=<admin password> \
     -e OGG_DEPLOYMENT=<deployment name> \
-    -e RUN_COMMAND=<script or command> \
-    -e RUN_COMMAND_BACKGROUND=<true | false> \
+    -e SETUP_USER_SCRIPTS=<path to setup scripts> \
+    -e STARTUP_USER_SCRIPTS=<path to startup scripts> \
+    -e ABORT_ON_USER_SCRIPT_ERRORS=<true | false>
```

Comment:
It might be better to handle errors from within the scripts. Perhaps if there was a generic `abortContainer.sh` command that terminated the container with an error message called from the user script?

Reply:
If I understand correctly, the abortContainer.sh would be expected to be called by the third party scripts as part of their error handling?  ie: their trap conditions would have to implement handing control to abortContainer.sh, which would then have to handle determining whether to abort or hand control back to the originating deployment-main.sh or continue with the existing thrid party script?

This seems like a pretty complex scripting model for a large majority of the user base to comprehend.

If simplification and avoidance confusion by the ABORT_ON_USER_SCRIPT_ERROR is the goal,  I would suggest the same as you -- any script non-zero exit aborts the boot; forcing the users to handle errors in their scripts.

Reply:
The `abortContainer.sh` script was just a thought and not something I expected to be part of this PR. IN the DBMS scripts (see `OracleDatabase/SingleInstance/dockerfiles/21.3.0/runUserScripts.sh`), there is no error handling.

Although I am not fond of that naive approach, I believe that whatever is implemented for OGG should be as similar as possible to what is already being done for the DBMS images. The reason for that belief is consistent implementation and behavior.